service/matching/taskQueueManager_test.go

```diff
@@ -573,8 +433,8 @@ func TestTaskQueueSubParitionSendsCurrentHashOfVersioningDataWhenFetching(t *tes
 				Return(asResp, nil)
 			tqm.matchingClient = mockMatchingClient
 		})
-	// Cram some versioning data in there so it will have something to hash when fetching
-	subTq.db.versioningData = data
+	subTq.dbTaskManager.Start()
+	subTq.dbTaskManager.taskQueueOwnership.setVersioningDataForNonRootPartition(data)
 	// Don't start it. Just explicitly call fetching function.
```

Comment:
This comment is now invalid as the task manager is indeed started. This probably works fine for the purposes of the test but the comment should be removed or changed.

Reply:
Added back.